swap/tests/happy_path.rs

```diff
@@ -35,8 +36,8 @@ async fn happy_path() {
     let xmr_alice = xmr_to_swap * 10;
     let xmr_bob = xmr_btc::monero::Amount::ZERO;
 
-    // todo: This should not be hardcoded
-    let alice_multiaddr: Multiaddr = "/ip4/127.0.0.1/tcp/9876"
+    let port = get_port().expect("Failed to find a free port");
+    let alice_multiaddr: Multiaddr = format!("/ip4/127.0.0.1/tcp/{}", port)
```

Comment:
🙃 Would potentially be nicer to let `init_alice` return the connection information she will use. That way we can change this code in one place, independent of each test.

Reply:
I think we did this or planned to do it #101